test/tools/rpc-client.js

```diff
@@ -29,4 +29,26 @@ describe('rpc client', () => {
       })
       .catch(done);
   });
+
+  it('should set a cursor key', done => {
```

Comment:
I'd love to see a test getting a cursor that doesn't exist, and setting an invalid store key (like `undefined` or `null` or a gigantic string), just so we can be sure failures are failing 😄

Reply:
yeah! getting a cursor that hasn't been set is fine and returns `null` (just like redis does). A store key missing is an error, and that behavior has a unit test on the server. I can go ahead and add something there though!